Formula/rethinkdb.rb

```diff
@@ -15,33 +15,12 @@ class Rethinkdb < Formula
   depends_on "boost" => :build
   depends_on "openssl@1.1"
 
-  # Fix error with Xcode 9, patch merged upstream:
-  # https://github.com/rethinkdb/rethinkdb/pull/6450
-  if DevelopmentTools.clang_build_version >= 900
-    patch do
-      url "https://raw.githubusercontent.com/Homebrew/formula-patches/fb00ee376a/rethinkdb/xcode9.patch"
-      sha256 "abd50d91a247ee7de988020dd9d405a3d4cd93edb2875b7d5822ba0f513f85a0"
-    end
-  end
-
-  # Upstream commit for OpenSSL 1.1 compatibility
-  patch do
-    url "https://github.com/rethinkdb/rethinkdb/commit/62456155.diff?full_index=1"
-    sha256 "6666074788d7de3295619426350c96b1f2e2e3b2427e76511dc168447034cacd"
-  end
-
-  # Upstream fix for Boost >= 1.69
-  patch do
-    url "https://github.com/rethinkdb/rethinkdb/commit/04785087.diff?full_index=1"
-    sha256 "a26e452ce1f16541a9ba40057af154e594fe89665c2c656994ceab103d2017e9"
-  end
-
   def install
     args = ["--prefix=#{prefix}"]
 
     # rethinkdb requires that protobuf be linked against libc++
     # but brew's protobuf is sometimes linked against libstdc++
-    args += ["--fetch", "protobuf"]
+    args += ["--fetch", "protobuf", "--allow-fetch"]
```

Comment:
From the configure doc:
> --allow-fetch           Allow fetching missing dependencies
This is not acceptable for Homebrew. We require to handle all dependencies outside of the build step to ensure reproducibility.

Reply:
Hello Moisan,
Thank you for your reply. I’ll handle that soon.